src/react_native/flat_list.cljs

```diff
@@ -39,3 +40,7 @@
 (defn flat-list
   [props]
   [react-native-flat-list (base-list-props props)])
+
+(defn flat-list-inside-gesture-handler
+  [props]
+  [gesture-handler/flat-list (base-list-props props)])
```

Comment:
flat-list is pretty complex component, and we have our own wrappers in cljs, so it won't work just like that